src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/index.jelly

```diff
@@ -230,6 +230,31 @@
                                         checked="${it.config.enableManualTrigger}"
                                         default="${true}"/>
                         </f:entry>
+                        <f:entry title="${%Fetch Project List Settings}"
+                                 help="/plugin/gerrit-trigger/help-GerritFetchProjectList.html">
```

Comment:
I would prefer we use standard jelly taglib functions as much as possible, since a lot of UI changes are coming. And too much custom html in plugin forms could get broken. Using the taglibs we should get those changes to be compatible "for free".

I think an ```optionalBlock``` here could work https://jenkins-ci.org/maven-site/jenkins-core/jelly-taglib-ref.html#form:optionalBlock